pydsa/dfs.py

```diff
@@ -0,0 +1,16 @@
+# Depth First Search
+# Complexity: O(V+E)
+#uses concept of list
+
+def dfsearch(graph,s):
```

Comment:
Add docstrings.

The function could be much simplified. See [bfs](https://github.com/pydsa/pydsa/blob/master/pydsa/bfs.py) for motivation.